bin/test-package.sh

```diff
@@ -21,4 +21,8 @@ source venv/bin/activate
 
 pip install $tarball
 
-python -c "import chromadb; api = chromadb.Client(); print(api.heartbeat())"
+if [[ $tarball == *"client"* ]]; then
```

Comment:
this conditional is brittle, lets make two different test shells and use them where required. we can pull the common out if wanted

Reply:
actually, I think its fine to just check the version tbh, this is a sanity to make sure the build worked